services/web/server/src/simcore_service_webserver/projects/projects_handlers_crud.py

```diff
@@ -147,6 +150,9 @@ async def create_projects(request: web.Request):
         query_params=query_params,
         predefined_project=predefined_project,
         product_name=req_ctx.product_name,
+        simcore_user_agent=request.headers.get(
```

Comment:
I need to make sure I do not forget tho add these headers in my PR

https://github.com/ITISFoundation/osparc-simcore/blob/bf8fcec519d02f4b03aaad04e2309907e30df443/api/specs/webserver/scripts/openapi_projects_crud.py#L62-L81